### ISSUE TITLE
Using $sheet in value field causes error

### ISSUE THREAD

**Issue by kyao** [usc-isi-i2/t2wml#162]
The error generated is:
```
0: Object { value: "'str' object has no attribute 'value'" }
```
The yaml file is:
```
statementMapping:
  region:
    - range: B3:B5
  template:
    item: =item[A, $row]
    property: =item[$col, 2, "property"]
    value: =value[$col, $row]
    qualifier:
    - calendar: Q1985727
      format: '%Y'
      precision: year
      property: P585
      time_zone: 0
      value: =$sheet
```

Data files:
[18_time.tar.gz](https://github.com/usc-isi-i2/t2wml/files/5082359/18_time.tar.gz)

**Comment by devowit** [comment]
fixed in t2wml-api version 0.0.10, and hence t2wml version 2.3.4
please close when tested ok

**Comment by kyao** [comment]
Tested ok using 2.3.4
